setup.py

```diff
@@ -78,7 +78,12 @@
         "Topic :: Database :: Front-Ends",
     ],
     python_requires='>=3.7',
-    install_requires=["pytz", "requests", "tzlocal"],
+    install_requires=[
+        "backports.zoneinfo;python_version<'3.9'",
```

Comment:
missing zoneinfo require should be a separate commit, it's not related to the fix itself.